lib/msal-node/src/cache/Storage.ts

```diff
@@ -166,37 +185,43 @@ export class Storage extends CacheManager {
      * @param inMemory
      */
     removeItem(key: string, type?: string): boolean {
+        this.logger.verbose("removeItem called");
         // read inMemoryCache
         const cache = this.getCache() as InMemoryCache;
         let result: boolean = false;
 
         // save the cacheItem
         switch (type) {
             case CacheSchemaType.ACCOUNT: {
+                this.logger.verbose(`Item type: ${CacheSchemaType.ACCOUNT}`);
```

Comment:
One alternative would be to log the type in the first log message in these functions.

Reply:
Good idea! Changed